src/leapYear.ts

```diff
@@ -1,3 +1,3 @@
-export const leapYear = (): boolean => {
-  return true;
+export const leapYear = (inputYear: number): boolean => {
+  return inputYear === 1996;
```

Comment:
In strict TDD, you dont need to return this conditional if you only have this one test
Just return true still as this is changing the behaviour of the code without a test driving you